tck/bridge-tck-main-portlet/src/main/java/com/liferay/faces/bridge/tck/tests/chapter_5/section_5_2/Tests.java

```diff
@@ -602,6 +602,41 @@ public PhaseId getPhaseId() {
 		return PhaseId.ANY_PHASE;
 	}
 
+	// Test is SingleRequest -- Render/Action
+	// Test #5.33 --
+	@BridgeTest(test = "headerPhaseListenerTest")
+	public String headerPhaseListenerTest(TestRunnerBean testRunner) {
```

Comment:
@stiemannkj1 Please send a follow-up PR after I merge. That might help you in your backport efforts as well.